contribs/drt/src/main/java/org/matsim/contrib/drt/run/DrtConfigGroup.java

```diff
@@ -323,6 +343,11 @@ protected void checkConsistency(Config config) {
 		if (useModeFilteredSubnetwork) {
 			DvrpModeRoutingNetworkModule.checkUseModeFilteredSubnetworkAllowed(config, mode);
 		}
+
+		if (simulationType == SimulationType.estimateAndTeleport) {
+			Verify.verify(drtSpeedUpParams == null, "Simulation type is estimateAndTeleport, but drtSpeedUpParams is set. " +
+				"Please remove drtSpeedUpParams from the config, as these two functionalities are not compatible.");
```

Comment:
See above, the two are exclusive. For the sake of consistency, maybe it also makes sense to rather activate estimate+teleport via the presence of the corresponding parameter set?

Reply:
We really prefer not to activate it implicitly. I also think this is not very good solution in drt speed up. 
As you suggested, adding a third enum would be a solution. However, this would require drt speed up users to set this new enum and effectively break their old config.  I don't have a strong opinion about this, I'm also ok with adding an additional enum.

Reply:
Fine for me